hca/added_command.py

```diff
@@ -217,37 +218,71 @@ def _build_non_body_payloads(cls, namespace):
         """
         endpoint_info = cls._get_endpoint_info()
         query_payload = {}
+        body_payload = {}
         header_payload = {}
         for (arg_name, arg) in namespace.items():
             # If the argument is positional, belongs in path, not a payload
             if arg_name not in endpoint_info["options"]:
                 continue
 
+            if arg_name in endpoint_info['body_params']:
+                # Check that it conforms to the right schema
+                try:
+                    jsonschema.validate(arg, endpoint_info['body_params'][arg_name])
+                except jsonschema.ValidationError as e:
+                    print(e)
```

Comment:
jsonschema validation or error printing?

Jsonschema validation is important because we want to make sure that objects being submitted through the python bindings are actually adhering to the format described in the swagger spec. 

Printing the exception probably isn't the best pattern, but a couple times it's helped me in debugging. I can probably find a better way to reformat that.